pkg/profilecreator/profilecreator.go

```diff
@@ -338,10 +340,11 @@ func (ghwHandler GHWHandler) getCPUsSequentially(reservedCPUCount int, htEnabled
 	}
 	totalCPUSet := totalCPUSetFromTopology(topologyInfoNodes)
 	isolatedCPUSet := totalCPUSet.Difference(reservedCPUSet.Result())
-	log.Infof("reservedCPUs: %v len(reservedCPUs): %d\n isolatedCPUs: %v len(isolatedCPUs): %d\n", reservedCPUSet.Result().String(), reservedCPUSet.Result().Size(), isolatedCPUSet.String(), isolatedCPUSet.Size())
+	log.Infof("%d reserved CPUs allocated: %v ", reservedCPUSet.Result().Size(), reservedCPUSet.Result().String())
+	log.Infof("%d isolated CPUs allocated: %v", isolatedCPUSet.Size(), isolatedCPUSet.String())
```

Comment:
maybe log once in the calling site? this could require some trivial refactoring

Reply:
Sure, will do that.